codegen/pubspec.yaml

```diff
@@ -1,8 +1,7 @@
 name: dart_code_gen
 description: A command line tool which generates Dart code for Protobuf type registries.
-version: 0.1.0
+version: 0.1.1
```

Comment:
Do we have a different versioning policy of this module compared to the version of the library (which is `0.1.3` AFAIU)?